src/App.vue

```diff
@@ -17,13 +27,20 @@ export default defineComponent({
     <NavBar />
   </header>
   <main>
-    <HomePage />
+    <MainView class="main-view" />
```

Comment:
The reason I created the `MainVIew` component is that I'm lazy importing all views in my router modules.
Is there a way to import the `<router-view />` component like below?
![image](https://user-images.githubusercontent.com/110823711/186170932-a4bc7f1e-767d-43eb-91c6-6a4c9ee6b223.png)

Reply:
As you are using destructuring assignment of `movieRoutes` in the `mainRoutes` they are a children of the `'/'` path. Vue router works this way that you don't need to pass `router-view` in nested routes if you don't want to nest components views inside each other. In your case the only one place where `router-view` is necessary is in `app.vue` component and you already passed it 👌 If you will remove importing `MainView.vue` from all your routes it will work as well as now. And also for now you are nesting one `MainView` in another and removing it from router also will help with this 😄

Reply:
Ok, got it now!
I'll commit the changes in a second 👍